composer.json

```diff
@@ -43,5 +44,20 @@
         "branch-alias": {
             "dev-master": "4.4-dev"
         }
-    }
+    },
+
+    "repositories": [
+        {
+            "type": "package",
+            "package": {
+                "name": "cucumber/cucumber",
+                "version": "dev-gherkin-16.0.0",
```

Comment:
for composer to work properly during a composer update, we need to ensure that update the reference for such a package repository is either updating the version of updating the `time` field for a dev version. Otherwise, composer will consider that your vendor folder is already uptodate (even though it does not have the same ref). Is this something corresponding to some tagged version ?

Reply:
This 'real' tag is `cucumber-gherkin/v16.0.0` but I couldn't seem to get that working

Reply:
```
Could not parse version constraint cucumber-gherkin/v16.0.0: Invalid version string "cucumber-gherkin/v16.0.0"
```

Reply:
that's indeed not a valid composer version

Reply:
Not sure how to handle it when that's the actual tag, any hints? 

https://github.com/cucumber/cucumber/releases/tag/cucumber-gherkin%2Fv16.0.0

Reply:
Well, as that's an internal thing, we don't care if composer considers that as a dev version rather than a stable version for min stability. And as we reference a tag, we *will* update the `version` field when changing the reference (as we will replace that `16.0.0` part). So the current way is OK.